components/aws/sagemaker/common/_utils.py

```diff
@@ -581,6 +592,208 @@ def get_best_training_job_and_hyperparameters(client, hpo_job_name):
     return best_job, train_hyperparameters
 
 
+def create_workteam(client, args):
+    """Create a workteam"""
+    with open('/app/common/workteam.template.yaml', 'r') as f:
+        request = yaml.safe_load(f)
+
+    request['WorkteamName'] = args['team_name']
+    request['Description'] = args['description']
+
+    if args['sns_topic']:
+    	request['NotificationConfiguration']['NotificationTopicArn'] = args['sns_topic']
+    else:
+        request.pop('NotificationConfiguration')
+
+    for group in [n.strip() for n in args['user_groups'].split(',')]:
+        request['MemberDefinitions'].append({'CognitoMemberDefinition': {'UserPool': args['user_pool'], 'UserGroup': group, 'ClientId': args['client_id']}})
+
+    for key, val in args['tags'].items():
+        request['Tags'].append({'Key': key, 'Value': val})
+
+    try:
+        response = client.create_workteam(**request)
+        portal = client.describe_workteam(WorkteamName=args['team_name'])['Workteam']['SubDomain']
+        logging.info("Labeling portal: " + portal)
+        return response['WorkteamArn']
+    except ClientError as e:
+        raise Exception(e.response['Error']['Message'])
+
+
+def create_labeling_job_request(args):
+    with open('/app/common/gt.template.yaml', 'r') as f:
+        request = yaml.safe_load(f)
+
+    algorithm_arn_map = {'us-west-2': '081040173940',
```

Comment:
maintenance will be hard for such things. I understand that there is no way as of now to poll such information. 
I would suggest please put some comment and add the source where you got these information from.